src/noise/CARMA_NoiseProcess.jl

```diff
@@ -0,0 +1,213 @@
+using FFTW
+using DiffEqNoiseProcess
+using LinearAlgebra: dot
+using Random
+using PyCall
+so = pyimport("scipy.optimize")
+
+@inline wiener_randn(rng::AbstractRNG,::Type{T}) where T = randn(rng,T)
+
+# == ARMAProcess: Generates ARMA Coefficients and Sinc-interpolated NoiseProcess == #
```

Comment:
Are there wikipedia or research paper links for how these noise processes are generated? Could we link to them in the code?